src/main/java/com/dabsquared/gitlabjenkins/listener/GitLabMergeRequestRunListener.java

```diff
@@ -35,7 +35,8 @@ public void onCompleted(Run<?, ?> build, @Nonnull TaskListener listener) {
             Integer mergeRequestId = cause.getData().getMergeRequestId();
             addNoteOnMergeRequestIfNecessary(build, trigger, listener, projectId.toString(), mergeRequestId, build.getParent().getDisplayName(), build.getNumber(),
                 buildUrl, getResultIcon(trigger, buildResult), buildResult.color.getDescription());
-            if (buildResult == Result.SUCCESS) {
+            if (buildResult == Result.SUCCESS
```

Comment:
I wouldn't expect that the option "Mark unstable builds as success in GitLab" has an effect on the option "Accept merge request on success".